playbook-website/app/views/layouts/docs.html.erb

```diff
@@ -16,7 +16,7 @@
           <% unless action_name == 'changelog' %>
             <%= pb_rails("background", props: {
               background_color: "dark",
-              image_url: @front_matter['header'] || asset_pack_url("media/images/pb_generic_header.jpg"),
+              image_url: @front_matter['header'] || asset_pack_url("media/images/getting-started.svg"),
```

Comment:
@Israel-Molestina Is pb_generic_header.jpg being called/used now? If not let's remove the image.